File: 프로그래머스/unrated/181887. 홀수 vs 짝수/홀수 vs 짝수.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> num_list) {
    int even,odd;
    for (int i = 0; i < num_list.size(); i++)
    {
        if (i % 2) odd += num_list[i];
        else even += num_list[i];
    }
    return max(odd,even);
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] arr) {
        int even = 0, odd = 0;
        for (int i = 1; i <= arr.length; i++) {
            if (i % 2 == 1) odd += arr[i - 1];
            else even += arr[i - 1];
        }
        return Math.max(even, odd);
    }
}